www/source/blog/2018-02-21-Habitat-Helm.html.md

```diff
@@ -106,7 +97,7 @@ nginx-1.11.10-20180221053714  0s
 
 ==> v1/Pod(related)
 NAME                                             READY  STATUS             RESTARTS  AGE
-honorary-wolf-habitat-operator-7f74859c6d-2jv9b  0/1    ContainerCreating  0         0s
+honorary-wolf-habitat-operator-7f74859c6d-2jv9b  1/1    ContainerCreating  0         0s
```

Comment:
When it's 1/1 the status is no `ContainerCreating` but `Running`.

Also think this should be nginx?

Reply:
no it really does use the `habitat-operator`, that's one issue with helm that I don't understand.